node_modules/.bin/bower

```diff
@@ -0,0 +1 @@
+../bower/bin/bower
```

Comment:
`node_modules` dependencies should be in `.gitignore`
with 5k+ file changes impossible to review code

Reply:
I'm done with it.
Branch already for review.